.github/workflows/installer.yml

```diff
@@ -13,19 +13,14 @@ jobs:
       fail-fast: false
       matrix:
         os: [macOS-latest, ubuntu-latest]
-        python-version: [3.6, 3.7]
+        python-version: [3.7, 3.8, 3.9]
 
     env:
       CI_OS: ${{ matrix.os }}
       PYVER: ${{ matrix.python-version }}
 
     steps:
     - uses: actions/checkout@v2
-      
-    - name: Wait for 2 hours if triggered by a release
-      if: (github.event_name == 'release') || (github.event_name == 'prerelease')
-      run:
-        sleep 7200
```

Comment:
Thanks for getting rid of this. That was always fun to run down!

Reply:
And to think we put that in with such good intentions 😢